mne_bids/write.py

```diff
@@ -1260,6 +1303,11 @@ def write_anat(bids_root, subject, t1w, session=None, acquisition=None,
             `theta`: is the angle of the defacing shear in degrees relative
                      to the normal to the plane passing through the anatomical
                      landmarks (default 35).
+    landmarks: instance of DigMontage | str
+        The DigMontage or filepath to a DigMontage with landmarks that can be
+        passed to provide information for defacing. Landmarks can be determined
+        from the head model using `mne coreg` GUI, or they can be determined
```

Comment:
You should always start from documentation before writing a single line of code. If the docstrings etc are clear, the code will be clear. Can you mention that the coordinate system of the digmontage must be 'mri'?